src/NodaTime.Demo/PeriodBuilderDemo.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2010 The Noda Time Authors. All rights reserved.
```

Comment:
Please change to 2019 :)

Reply:
Sure, want me to update the date on any other files I touch?

Reply:
Nope, just new ones.